helpers.js

```diff
@@ -33,6 +34,20 @@ const specNameMap = {
     Windwalker: 'WW'
 };
 
+const sayQuotes = [
+    'Many times have I fallen at the hands of an orc and many times have I risen again.',
+    'For every orc that struck me, I will cleave a thousand of their skulls! This ends here!',
+    'For the Horde, my friend.',
+    'I may be bruised and beaten, but the hatred boils inside me.'
+];
+
+const shoutQuotes = [
+    'THIS ENDS HERE!',
+    `<:gamon_shout:${GAMON_SHOUT_ID}> I, GAMON, WILL SAVE US!`,
+    ':axe: TASTE MY AXE!',
```

Comment:
Iconic axe in game emoji? I don't know my axes but there's got to be one

Reply:
Picked Gorehowl, since it's the first one that came to mind haha

Reply:
👍🏼